src/main/java/duke/exceptions/InvalidCommandException.java

```diff
@@ -0,0 +1,11 @@
+package duke.exceptions;
+
+/** Exception that is thrown when an invalid command is given. */
+public class InvalidCommandException extends Exception {
+   
+    private static final long serialVersionUID = 1L;
+
```

Comment:
static final variables should be all caps, with '_' to represent spacing. Problem is consistent with next few classes

Reply:
serialVersionUID  is a field defined by the abstract class Serializable, which is a java import. I'm unfortunately unable to change the naming convention for this field,